src/ucp/api/ucp.h

```diff
@@ -591,6 +591,9 @@ enum ucp_send_am_flags {
     UCP_AM_SEND_FLAG_RNDV  = UCS_BIT(2),             /**< Force UCP to use only
                                                           rendezvous protocol for
                                                           AM sends. */
+    UCP_AM_SEND_FLAG_COPY_HEADER  = UCS_BIT(3),      /**< Valid user header during send 
```

Comment:
Please align `=` on column

Reply:
I think `COPY` is not a good word here, because UCX doesn't really always copy it.
Maybe something like: `TEMP_USER_HEADER`, or something else. WDYT?

Reply:
You are right but the user don't know it... 
It's maybe make us look "bad" from the user perspective, and this is a good reason to pick a better name if this is what you meant

Reply:
But maybe this is a good reason to find a better name, we always copy it so it's not really a good name...
It's not describe the functionality correctly.

Reply:
Maybe UCP_AM_SEND_FLAG_TEMP_HEADER
This signal UCX that this header is temporary and valid only for the duration of the send function?